contracts/Transfer.sol

```diff
@@ -142,24 +142,24 @@ contract Transfer is FraudProofHelpers {
 
     // BLS PROOF VALIDITY IMPL END
 
-    // /*********************
-    //  * Constructor *
-    //  ********************/
-    // constructor(address _registryAddr) public {
-    //     nameRegistry = Registry(_registryAddr);
-
-    //     governance = Governance(
-    //         nameRegistry.getContractDetails(ParamManager.Governance())
-    //     );
-
-    //     merkleUtils = MTUtils(
-    //         nameRegistry.getContractDetails(ParamManager.MERKLE_UTILS())
-    //     );
-
-    //     tokenRegistry = ITokenRegistry(
-    //         nameRegistry.getContractDetails(ParamManager.TOKEN_REGISTRY())
-    //     );
-    // }
+    /*********************
+     * Constructor *
+     ********************/
+    constructor(address _registryAddr) public {
+        nameRegistry = Registry(_registryAddr);
+
+        governance = Governance(
+            nameRegistry.getContractDetails(ParamManager.Governance())
+        );
+
+        merkleUtils = MTUtils(
```

Comment:
If you can't pass the fraudproof test, one of the reasons might be the merkleUtils here is disabled before.